examples/inventory_grid_stacked_transfer.gd

```diff
@@ -40,6 +40,13 @@ func _input(event: InputEvent) -> void:
 
     lbl_info.set_global_position(get_global_mouse_position() + info_offset)
 
+func _can_drop_data(at_position: Vector2, data: Variant) -> bool:
+    return true
+
+func _drop_data(at_position: Vector2, data: Variant) -> void:
+    ctrl_inventory_left.inventory.remove_item(data.item)
+    ctrl_inventory_right.inventory.remove_item(data.item)
```

Comment:
Ugh, I noticed one more issue with this:
After the item is dropped and removed from the inventory it is not being freed. And since we have no custom logic for handling the item drop in the examples, this will result in orphan nodes (i.e. memory leak). Not a big deal when it comes to examples, but it could mislead some users who might think that items don't need to be freed after being removed from the inventory.
I suggest adding a `queue_free` call and modifying the comment:
```gdscript
# Replace the following line with custom logic for handling the item drop:
data.item.queue_free()
```

Reply:
Great catch! I didn't notice, and I even was already using this in my project without freeing the item itself lol :sweat_smile: